doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1005,7 +1005,162 @@ Response code:
             "temperature"
         ]
 
-## Standard OMA-NGSI operations [/v2/queryContext]
+## Context Subscriptions [/v2/entities/{entityId}/subscriptions]
+
+This section is still under discussion.
```

Comment:
Once we finish this PR maybe this sentence could be removed (to be homogeneous with other sections, which don't include it). There is a global comment at the beginning of the document saying that all is yet under discussion and I think that with that is enough.